chapters/13-package-release.Rmd

```diff
@@ -10,4 +10,47 @@
 
 <!-- NOTE: Preferably something linked to working towards completing the project assignments, maybe to be used as a milestone? -->
 
+The purpose of this exercise is for the participant to go through an iteration of doing an R CMD check of the package in remote systems, understanding how to use these systems and lastly practice how to tag git commits to track versions.
 
+A. Check builds on other systems (with rhub)
+<!--> https://r-hub.github.io/rhub/articles/rhub.html-->
+
+1. Validate email with rhub
+> `rhub::validate_email("youremail@email.com")`
+
+2. Run a package check against the directory
+> `rhub::check()`
+
+3. Run a package check against the tarball
+> `devtools::build()`
+> `rhub::check(<tarball>)`
+
+4. Run a package check against a specific OS
+> `rhub::check(<OS>)`
+
+5. Browse previous checks
+> `rhub::list_package_checks()`
+
+B. Etiquette for using (mostly free) servers and external build systems like GitHub Actions and rhub (e.g. do not overuse them)
+
+1. Read the usage limits on GitHub Actions and rhub
+> - https://docs.github.com/en/actions/reference/usage-limits-billing-and-administration
+> - https://docs.r-hub.io/#regularly
+
+1(b). Run the GitHub Actions for the package, locally (optional)
+> https://github.com/nektos/act
+
+1(c). Run the rhub check locally (optional)
+> https://r-hub.github.io/rhub/articles/local-debugging.html
```

Comment:
I'm not really familiar with these, are they hard to do?  Perhaps that is why they are optional?

Reply:
They aren't hard to do. They are all implemented really nicely in the rhub package. Maybe the only challenge might be working with docker, which rhub seems to do. Optional is a good choice for now, we can revisit later.